modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/info_bar/end.jsp

```diff
@@ -28,10 +28,29 @@
 </div>
 
 <c:if test="<%= fixed %>">
-	<aui:script require="metal-affix/src/Affix">
-		new metalAffixSrcAffix.default({
-			element: '.info-bar-container',
-			offsetTop: 15,
-		});
+	<aui:script sandbox="<%= true %>">
+		const element = document.querySelector('.info-bar-container');
+
+		let lastPosition;
+		let topOffset = 15;
+
+		function checkPosition() {
+			let newPosition = 'affix';
+
+			if (document.defaultView.pageYOffset <= topOffset) {
+				newPosition = 'affix-top';
+			}
+
+			if (element && newPosition !== lastPosition) {
+				element.classList.add(newPosition);
+				element.classList.remove(lastPosition);
+
+				lastPosition = newPosition;
+			}
+		}
+
+		checkPosition();
+
+		document.addEventListener('scroll', checkPosition);
```

Comment:
Looking at [metal-affix](https://github.com/deprecate/metal-affix/blob/master/src/Affix.js#L40-L42), we are defining this event on `elementChanged`, `offsetTopChanged` and `offsetBottomChanged`. Is it safe to ignore these?

Reply:
Since this use case is _only_ for affixing this single element to the top and has the same element and offset, there is no need to listen for those. Those are included in metal-affix in case the element or offset values change.